pcmanfm/connectserverdialog.cpp

```diff
@@ -41,11 +41,11 @@ QString ConnectServerDialog::uriText() {
   uri = QString::fromLatin1(serverType.scheme);
   uri += QStringLiteral("://");
   if(ui.loginAsUser->isChecked()) {
-    uri += ui.userName->text();
+    uri += ui.userName->text().trimmed();
     uri += QLatin1Char('@');
   }
 
-  uri += ui.host->text();
+  uri += ui.host->text().trimmed();
```

Comment:
Since you trim the texts of `ui.userName` and `ui.host`, please also consider that inside the method `ConnectServerDialog::checkInput()`, by replacing `ui.userName->text().isEmpty()` and `ui.host->text().isEmpty()` with `ui.userName->text().trimmed().isEmpty()` and `ui.host->text().trimmed().isEmpty()` respectively.

Reply:
Sorry for that. I've updated it.